samples/durable-client-managed-identity/functions-app/DurableClientSampleFunctionApp.csproj

```
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <AzureFunctionsVersion>v3</AzureFunctionsVersion>
```

These should all be updated:


```suggestion
    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
```

---

Done